lib/turbo-rails.rb

```diff
@@ -1,4 +1,5 @@
 require "turbo/engine"
+require 'active_support/core_ext/module/attribute_accessors_per_thread'
```

Comment:
Prefer double quotes in Ruby code.

```suggestion
require "active_support/core_ext/module/attribute_accessors_per_thread"
```